### ISSUE TITLE
Network Error when using privacy badger

### ISSUE THREAD

**Issue by ligi** [balance-io/balance-manager#379]
I can only use the balance manager with firefox or brave - getting network error with chromium all the time:
![image](https://user-images.githubusercontent.com/111600/43410034-d8a66d2e-9425-11e8-840b-77401a3b2675.png)

**Comment by ricburton** [comment]
@jinchung says it looks like it's related to cryptocompare

**Comment by ricburton** [comment]
Given that it's working in other browsers, could it be related to an extension you have in Chrome @ligi?

**Comment by ligi** [comment]
this is possible - let me check.

**Comment by ligi** [comment]
yes - it is the privacy badger - so you should be able to reproduce the problem when installing: https://www.eff.org/privacybadger

**Comment by ricburton** [comment]
Thanks @ligi. Given our customer base, we should detect for that and explain why we need access to this API.

That would be better than it just failing with Network Error.

**Comment by ligi** [comment]
Or better contact cryptocompare to fix the problem. You can use API's with privacy badger - but somehow it detects this api-usage as problematic.

**Comment by ricburton** [comment]
We should have links to a status page for our data providers (and our servers).

We should also have a list of chrome extensions (are they detectable) that might be causing the proble.

We should suggest that a customer tries another browser before assuming that our software is completely broken :)

I will think on this. It should all be added into the work in issue #300.

**Comment by ricburton** [comment]
Yeah, @vcealicu, how can we better handle this?
